src/Lunapark/Endpoint.purs

```diff
@@ -169,23 +170,27 @@ printPart = case _ of
   Flick → "flick"
 
 handleAPIError
-  ∷ N.Response (Either N.ResponseFormatError Json)
+  ∷ Either N.Error (N.Response Json)
```

Comment:
Would you mind hide this function from export in your pr? Apparently `Either a b -> Either c d` seems a bit too `bimap` to be exported 😀

Reply:
done